specifyweb/frontend/js_src/lib/components/Errors/FormatError.tsx

```diff
@@ -111,8 +112,7 @@ export function formatError(
 /** Format error message as JSON, HTML or plain text */
 function formatErrorResponse(error: string): JSX.Element {
   try {
-    const json = JSON.parse(error);
-    return <pre>{jsonStringify(json, 2)}</pre>;
+    return formatJSONBackendResponse(error);
```

Comment:
Could you rename this to `formatJsonBackendResponse` to follow the strictCamelCase convention? Don't want to do what Internet Explorer did with `XMLHttpRequest`